### PR TITLE
Pp 615 skonto transfer summary integration tests

### PR DESCRIPTION
**Pull request by mrkulik**

### CODE REVIEW

File: BankSDK/GiniBankSDKExample/Tests/IntegrationTests/TransferSummary/SkontoTransferSummaryIntegrationTest.swift [hunk 1]
```diff
@@ -0,0 +1,222 @@
...
+        
+        GiniBankConfiguration.shared.skontoDiscounts = [modifiedSkontoExtractions]
+    }
+
+    override func sendTransferSummary(result: AnalysisResult) {
+        guard let updatedAmountToPayString = result.extractions["amountToPay"]?.value else { return }
+        let amountExtraction = createAmountExtraction(value: updatedAmountToPayString)
+        GiniBankConfiguration.shared.sendTransferSummaryWithSkonto(amountToPayExtraction: amountExtraction,
+                                                                   amountToPayString: updatedAmountToPayString)
+    }
```
Comment (ValentinaIancu-Gini) [inline BankSDK/GiniBankSDKExample/Tests/IntegrationTests/TransferSummary/SkontoTransferSummaryIntegrationTest.swift:206]: [clarification] Why do we do this here:
`  guard let updatedAmountToPayString = result.extractions["amountToPay"]?.value else { return }`  and in the same method but for `SkontoNotAppliedTransferSummaryHandler` we don't do this?
If there is no difference can we have just one?

**Comment by ValentinaIancu-Gini** [comment]
@mrkulik Really nice job with these tests!

File: BankSDK/GiniBankSDKExample/Tests/IntegrationTests/TransferSummary/SkontoTransferSummaryIntegrationTest.swift (hunk 1 above)
Comment (mrkulik) [reply BankSDK/GiniBankSDKExample/Tests/IntegrationTests/TransferSummary/SkontoTransferSummaryIntegrationTest.swift:206]: @ValentinaIancu-Gini thats true, I've checked that after some changes (during development) I have same overrides for both specific handlers and they are the same as default implementation in base handler. I removed this and left only base implementation for both, thank you for raising it!

**Review by ValentinaIancu-Gini — APPROVED** [review]
@mrkulik Thank you!

**Pull request merged**